frontend/packages/db-structure/src/parser/sql/postgresql/converter.ts

```diff
@@ -246,6 +311,55 @@ export const convertToSchema = (stmts: RawStmt[]): ProcessResult => {
     comment: string | null
   }
 
+  // biome-ignore lint/complexity/noExcessiveCognitiveComplexity: <explanation>
```

Comment:
The index of complexity of this function is 16 while the threshold is 15. I gave up to reduce it to matches the threshold 😅 

I think the parsers' functions tend to be more complex than the modules of other packages. It might be possible to make the threshold higher for this package 🤔

Reply:
You are right, it would be difficult to be compliant with a parser. Can you create a PR to change the settings?